content/en/docs/setup/install/multicluster/shared/index.md

```diff
@@ -165,27 +209,51 @@ spec:
 EOF
 {{< /text >}}
 
+{{< /tab >}}
+
+{{< /tabset >}}
+
 Apply the main cluster's configuration.
 
 {{< text bash >}}
 $ istioctl --context=${MAIN_CLUSTER_CTX} manifest apply -f istio-main-cluster.yaml
 {{< /text >}}
 
-Wait for the control plane to be ready before proceeding. Set the `ISTIOD_REMOTE_EP` environment
-variable based on which remote control plane configuration option was selected earlier:
+Wait for the control plane to be ready before proceeding.
+
+{{< text bash >}}
+$ kubectl --context=${MAIN_CLUSTER_CTX} -n istio-system get pod
+NAME                                    READY   STATUS    RESTARTS   AGE
+istio-ingressgateway-7c8dd65766-lv9ck   1/1     Running   0          136m
+istiod-f756bbfc4-thkmk                  1/1     Running   0          136m
+prometheus-b54c6f66b-q8hbt              2/2     Running   0          136m
+{{< /text >}}
+
+Set the `ISTIOD_REMOTE_EP` environment variable based on which remote control
+plane configuration option was selected earlier.
+
+{{< tabset category-name="platform" >}}
 
-* Option (1) - `istio-ingressgateway` gateway shared with data traffic
+{{< tab name="istio-ingressgateway" category-value="istio-ingressgateway" >}}
 
 {{< text bash >}}
 $ export ISTIOD_REMOTE_EP=$(kubectl --context=${MAIN_CLUSTER_CTX} -n istio-system get svc istio-ingressgateway -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+$ echo "ISTIOD_REMOTE_EP is ${ISTIOD_REMOTE_EP}"
 {{< /text >}}
 
-* Option (2) - Use a cloud provider’s internal load balancer on the Istiod service.
+{{< /tab >}}
+
+{{< tab name="Internal Load Balancer" category-value="internal-load-balancer" >}}
 
 {{< text bash >}}
 $ export ISTIOD_REMOTE_EP=$(kubectl --context=${MAIN_CLUSTER_CTX}  -n istio-system get svc istiod -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
```

Comment:
Nitpick: This has two spaces before the "-n"; the other tab has only one space before the "-n". But fine to leave alone.